Source/ACE/Entity/Landblock.cs

```diff
@@ -104,10 +104,13 @@ public Landblock(LandblockId id)
             var factoryObjects = WorldObjectFactory.CreateWorldObjects(objects);
             factoryObjects.ForEach(fo =>
             {
-                if (!worldObjects.ContainsKey(fo.Guid))
+                if (fo != null)
```

Comment:
technically impossible in the current state of the code. I added the null check when while I was temping in the removal of GenericObject to reconfirm what I already knew. WOF.CreateWorldObjects prevents nulls in results now so the fo null check is just not possible to hit ever, and technically so is the null in CreateWorldObjects as long as we set a default object, which I see no reason no to do.